src/SwedbankPay.Sdk/PaymentInstruments/Invoice/IInvoicePaymentsResource.cs

```diff
@@ -1,25 +1,25 @@
-﻿using System;
-using System.Threading.Tasks;
-
-namespace SwedbankPay.Sdk.Payments.InvoicePayments
-{
-    public interface IInvoicePaymentsResource
-    {
-        /// <summary>
-        ///     Creates a new invoice payment
-        /// </summary>
-        /// <param name="paymentRequest"></param>
-        /// <param name="paymentExpand"></param>
-        /// <returns></returns>
-        Task<InvoicePayment> Create(InvoicePaymentRequest paymentRequest, PaymentExpand paymentExpand = PaymentExpand.None);
-
-
-        ///// <summary>
-        /////     Gets an existing invoice payment.
-        ///// </summary>
-        ///// <param name="id"></param>
-        ///// <param name="paymentExpand"></param>
-        ///// <returns></returns>
-        Task<InvoicePayment> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None);
-    }
+﻿using System;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Payments.InvoicePayments
+{
+    public interface IInvoicePaymentsResource
+    {
+        /// <summary>
+        ///     Creates a new invoice payment
+        /// </summary>
+        /// <param name="paymentRequest"></param>
+        /// <param name="paymentExpand"></param>
+        /// <returns></returns>
+        Task<IInvoicePayment> Create(InvoicePaymentRequest paymentRequest, PaymentExpand paymentExpand = PaymentExpand.None);
+
+
+        ///// <summary>
+        /////     Gets an existing invoice payment.
+        ///// </summary>
+        ///// <param name="id"></param>
+        ///// <param name="paymentExpand"></param>
+        ///// <returns></returns>
+        Task<IInvoicePayment> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None);

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)